appyters/Researcher_Summary_Report_Appyter/researcher_summary_appyter.ipynb

```diff
@@ -0,0 +1,746 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%appyter init\n",
+    "from appyter import magic\n",
+    "magic.init(lambda _=globals: _())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter hide_code_exec\n",
+    "{% do SectionField(\n",
+    "    name='Data_Section1',\n",
+    "    title='Input the name of Investigator and the ORCID ID as an optional input if known'\n",
+    ") %}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter code_exec\n",
+    "{% set researcher_name = StringField(\n",
+    "    name='researcher_name', \n",
+    "    label='Investigator Name', \n",
+    "    default=\"Robert J. Lefkowitz\", \n",
+    "    description='Input full name of PI along with middle names/middle initials followed by spaces as leaving it out may change the results of certain information', \n",
+    "    section='Data_Section1', \n",
+    "    required = True\n",
+    "    )\n",
+    "%}\n",
+    "{% set orcid = StringField(\n",
+    "    constraint= \"[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{4}|.{0}|[0-9]{4}\",\n",
+    "    name='researcher_id', \n",
+    "    label='(Optional) Investigator Orcid ID if Known', \n",
+    "    default='', \n",
+    "    description='Input the ORCID associated with the researcher if known in addition to the inputted name.', \n",
+    "    section='Data_Section1'\n",
+    "    )\n",
+    "%}\n",
+    "researcher_name = {{researcher_name}}\n",
+    "orcid_id = {{orcid}}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "split_name = researcher_name.split()\n",
+    "name_of_researcher = researcher_name.split()[0] + \" \" + researcher_name.split()[-1]\n",
+    "pubmed_name = split_name[-1] + \" \" +split_name[0][0]\n",
+    "if len(split_name) > 2:\n",
+    "    pubmed_name = split_name[-1] + \" \"\n",
+    "    for name in split_name[:-1]:\n",
+    "        pubmed_name += name[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
`%%appyter markdown` cells must be code cells to function properly

Reply:
Changed all the markdown cells to code cells now.